Pipfile

```diff
@@ -38,6 +38,8 @@ pglast = "*"
 async-lru = "*"
 httpx = "*"
 pendulum = "*"
+aioboto3 = "*"
+typing-extensions = "==3.7.4.3"
```

Comment:
Do you really need this?
Sounds like this is a backport of typing for python 3.5/ 3.6

_Users of other Python versions should continue to install and use the typing module from PyPi instead of using this one unless specifically writing code that must be compatible with multiple Python versions or requires experimental types._

Reply:
It was needed for aiboto3 to install properly for some reason, otherwise it would give complaints that it couldn't find it. Their PyPi says they support 3.8, but I guess still have dependencies on this library.